### DISCUSSION TITLE
Feature Request: Share link to other apps

### DISCUSSION THREAD

**Discussion by lekshmanmj** [flet-dev/flet#5497] [Extensions]
**Requirement:**
It would be better if flet has a feature to share links to other apps.

**Reference Links:**
1. https://pub.dev/documentation/share/latest/
2. https://pub.dev/packages/share_plus

**Expectation**
https://miro.medium.com/v2/resize:fit:896/1*57ZF5y6b8Esr6BNDIQGDxw.jpeg

**Comment by jiho-m0521** [comment]
True. It would be awesome to add this feature

**Comment by Muddassir-Farooq-official** [comment]
Any progress
